project/build/dev/d8/themes/custom/patternlab/source/_meta/_00-head.twig

```diff
@@ -15,6 +15,7 @@
   <link rel="stylesheet" href="../../../../dest/style.css">
   <!-- endinject -->
   <link rel="stylesheet" href="../../../../source/styleguide/custom-styleguide-specific.css">
-
+  <script src="https://use.typekit.net/jhz5ozy.js"></script>
```

Comment:
You mentioned earlier that you were worried TypeKit would bring over unnecessary fonts. This TypeKit id therefore needs to be updated through the TypeKit UI to not bring over those font faces.